ampligraph/latent_features/models/ComplEx.py

```diff
@@ -1,4 +1,6 @@
-from . import EmbeddingModel
+from .EmbeddingModel import EmbeddingModel
+from ampligraph.latent_features import constants as CONSTANTS
```

Comment:
No major issue, but I think CONSTANTS should just be left lowercase, it's fairly clear what it is either way.